capitolwords_ng/capitolweb/legislators/urls.py

```diff
@@ -3,6 +3,8 @@
 from . import views
 
 urlpatterns = [
-    url(r'^person/(?P<name>\w+)/$', views.find_by_name, name='search_by_speaker')
-
+    url(r'^person/(?P<name>\w+)/$', views.find_by_name, name='search_by_speaker'),
```

Comment:
The primary key is just a db increment. There are a bunch of externalIDs, bioguide being one of them, and they're stored in a different table with a foreign key to the CongressPerson table. we could switch this up and have bioguide be the default for the rest endpoint.